cpp/src/Slice/Parser.h

```diff
@@ -702,16 +705,17 @@ class Operation : public virtual Contained, public virtual Container
     size_t returnBitSequenceSize() const;
 
     void destroy() override;
-    TypePtr returnType() const;
-    bool returnIsTagged() const;
-    int returnTag() const;
+    TypePtr returnType() const; //TODO remove this once the compilers have been updated to use return-tuples.
+    bool returnIsTagged() const; //TODO remove this once the compilers have been updated to use return-tuples.
+    int returnTag() const; //TODO remove this once the compilers have been updated to use return-tuples.
     Mode mode() const;
     Mode sendMode() const;
     bool hasMarshaledResult() const;
     MemberPtr createParameter(const std::string&, const TypePtr&, bool, bool, int);
```

Comment:
Would be nice to give names to these function parameters.

Reply:
Yes, I was planning on doing it for all the Slice header files in one go, instead of doing it in pieces. I find it tends to be more efficient that way.